aiobotocore/session.py

```diff
@@ -37,6 +42,7 @@ def __init__(
         event_hooks=None,
         include_builtin_handlers=True,
         profile=None,
+        load_executor=False,
```

Comment:
instead of this being a bool, you should just pass in the executor you want to use for the loader, that way you avoid defaulting to creating a ton of threads